subprojects/docs/src/main/resources/header.html

```diff
@@ -95,6 +95,7 @@
             </div></li>
             <li><a href="/release-notes.html">Release Notes</a></li>
             <li><a href="/userguide/installation.html">Installing Gradle</a></li>
+            <li><a href="/userguide/getting_started.html">Getting Started</a></li>
```

Comment:
The top left nav now is getting quite cluttered. I think some of these links could go below and we can break this up a bit.

Reply:
How about this?

![screenshot 2018-09-13 17 34 24](https://user-images.githubusercontent.com/51534/45522980-5afa1780-b77b-11e8-946e-4782cbeff9c2.png)

Reply:
Note that I had to do a little bit of CSS trickery to make a gap. I'm happy to handle the nav separately if you like.

Reply:
@eriwen Looking at the proposal again, where would release notes go?

I'm OK with losing the "User Manual Home" link as long as a _Documentation_ link is retained in the top nav. I see that on the nightly docs, there is no such link at all, not even the old drop-down list.